addons/mail/static/src/messaging/component/message/message_tests.js

```diff
@@ -143,18 +137,17 @@ QUnit.test('delete attachment linked to message', async function (assert) {
         body: "<p>Test</p>",
         id: 100,
     });
-    await this.createMessage(messageLocalId);
+    await this.createMessageComponent(message);
     document.querySelector('.o_Attachment_asideItemUnlink').click();
     await afterNextRender();
-    assert.notOk(this.env.store.state.attachments['ir.attachment_10']);
-    assert.notOk(this.env.store.state.messages[messageLocalId].attachmentLocalIds['ir.attachment_10']);
```

Comment:
is there not a way to check the attachment is linked to the message as before?

Reply:
I've removed this because component tests should not inspect store